src/pages/get-involved/join-our-team.tsx

```diff
@@ -25,16 +48,124 @@ export default function JoinOurTeamPage({
 
   return (
     <>
+      {/* Header */}
+      <Section backgroundColor="tertiary.light">
+        <Grid container justifyContent="space-between">
+          {/* Header text + button */}
+          <Grid xs={12} md={6}>
+            <Stack spacing={6} alignItems={{ xs: "center", md: "self-start" }} padding={{ xs: 2, lg: 12 }}>
+              <Typography variant="h3" color="black" textAlign={{ xs: "center", md: "left" }}>
+                {sanityJoinOurTeamPage?.mainHeader}
+              </Typography>
+              <Typography variant="h6" color="black" textAlign={{ xs: "center", md: "left" }}>
+                {sanityJoinOurTeamPage?.subHeader}
+              </Typography>
+              <SanityButton
+              isAnimated
+              boopProps={{ scale: 1.1 }}
+              content={sanityJoinOurTeamPage.headerButton}>
+              </SanityButton>
+            </Stack>
+          </Grid>
+          <Grid md={6} sx={{ display:{ xs: "none", md: "flex" } }} justifyContent="center" alignItems="center">
```

Comment:
nit: spacing here between `display: {`. Same kind of thing that prettier would fix